tracking/translations_parser/cli/taskcluster.py

```diff
@@ -82,6 +82,24 @@ def get_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
+def is_running_in_ci():
```

Comment:
@bhearsum you probably discussed this with Greg already but I'm curious if it's the only way to determine that we're in ci